pyproject.toml

```diff
@@ -112,6 +112,12 @@ select = [
   "W",
   "YTT",
 ]
+# Add type checking codes
+extend-select = [
```

Comment:
Why not adding these to the "select" block above?

Reply:
thought its better style to group all the type annotation stuff but adding it back